package.json

```diff
@@ -36,6 +36,8 @@
     "run": "jpm run --addon-dir dist/ ${npm_config_jpm_runflags} --prefs jpm-prefs.json",
     "pretest": "npm run-script autolint",
     "test": "cross-env NODE_ENV=test karma start",
+    "preintegration": "cross-env ENABLE_DOORHANGER=1 npm run package",
```

Comment:
Just as a sanity check, do we still need this ENV var magic?
I thought we yanked all the `ENABLE_DOORHANGER` in PR #403; "Remove ENABLE_DOORHANGER code".

Reply:
will remove; when first submitted this was still necessary.